src/searchdialog.cpp

```diff
@@ -29,10 +29,13 @@
 #include <QProgressDialog>
 #include <QSignalBlocker>
 
-
+/**
```

Comment:
docstring comments are normally placed in header and not in implementation. Also formatting in many places git broken...